packages/experimental/TabList/src/Tab/useTabAnimation.ts

```diff
@@ -58,24 +58,30 @@ export function useTabAnimation(
             e.nativeEvent.layout.height <= layout.tablist.height &&
             e.nativeEvent.layout.height < RENDERING_HEIGHT_LIMIT))
       ) {
-        let width: number, height: number;
+        const { width: tabWidth, height: tabHeight, y: tabY } = e.nativeEvent.layout;
+        let indicatorWidth: number, indicatorHeight: number;
         // Total Indicator inset consists of the horizontal/vertical margin of the indicator, the space taken up by the tab's focus border, and the
         // existing padding between the focus border and the tab itself. Multiply by 2 to account for the start + end margin/border/padding.
         const focusBorderPadding = 1;
         const totalIndicatorInset = 2 * (tokens.indicatorMargin + tokens.borderWidth + focusBorderPadding);
         // we can calculate the dimensions of the indicator using token values we have access to.
         if (vertical) {
-          width = tokens.indicatorThickness;
-          height = e.nativeEvent.layout.height - totalIndicatorInset;
+          indicatorWidth = tokens.indicatorThickness;
+          indicatorHeight = tabHeight - totalIndicatorInset;
         } else {
-          width = e.nativeEvent.layout.width - totalIndicatorInset;
-          height = tokens.indicatorThickness;
+          indicatorWidth = tabWidth - totalIndicatorInset;
+          indicatorHeight = tokens.indicatorThickness;
+        }
+        let tabX = e.nativeEvent.layout.x;
+        // For RTL users, we adjust the x position of each tab to be relative to the entire tablist starting right to left
+        if (Platform.OS == ('win32' as any) && I18nManager.isRTL) {
+          tabX = layout.tablist.width - (tabX + tabWidth);
```

Comment:
Kind of surprised you have to do this considering that x eventually gets set to the start style property, which should take RTL into account. am I misreading the code?

Reply:
The intuitive fix would initially be to just use `left` instead of `start` for the given x, but on both mac and win32, setting `left: x` when RTL is enabled, moves the transform origin towards the right side of the tablist rather than the left. This makes the indicator fly way off screen.